migration/v0.42/apply/10-upgrade-grafana.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+ROOT="$(readlink -f "$(dirname "${0}")/../../../")"
+
+# shellcheck source=scripts/migration/lib.sh
+source "${ROOT}/scripts/migration/lib.sh"
+
+run() {
+  case "${1:-}" in
+  execute)
+    if [[ "${CK8S_CLUSTER}" =~ ^(sc|both)$ ]]; then
+      log_info "Checking current Grafana release versions"
+
+      ops_version=$(helm_do sc get metadata -n monitoring ops-grafana -ojson | jq '.version' | tr -d '"')
+      user_version=$(helm_do sc get metadata -n monitoring user-grafana -ojson | jq '.version' | tr -d '"')
+
+      if [[ "${ops_version}" < "8.4.7" || "${user_version}" < "8.4.7" ]]; then
```

Comment:
Does the `<` and `>` operators work as lexicographic checks in `[[ ]]`?

Reply:
[Yes](https://www.gnu.org/savannah-checkouts/gnu/bash/manual/bash.html#index-_005b_005b), so since we are upgrading from Grafana with a major chart version of `7` I don't think there should be any issues with this expression. I have tested running the script after having upgraded to e.g. apps v0.40.1 where Grafana is on chart version `7.3.12`.